renderers/standings.py

```diff
@@ -32,3 +38,26 @@ def render(matrix, canvas, division):
 
     canvas.Fill(*ledcolors.standings.fill)
     stat = 'w' if stat == 'l' else 'l'
+
+def render_static_wide_standings(matrix, canvas, division, font, text_color):
+  while True:
+    offset = 6
+
+    for team in division.teams:
+      team_text = team.team_abbrev
+      graphics.DrawText(canvas, font, 1, offset, text_color, team_text)
+
+      team_record = str(team.w) + "-" + str(team.l)
+      stat_text = '{:6s} {:4s}'.format(team_record, str(team.gb))

```

Comment:
Does this handle decimals? It's common to have a team be 3.5 games back or something similar. Should be able to test easily by changing the date passed in main.py

Reply:
Yes. The gb attribute returns - for the division leader and then a 1.0, 2.5, etc for the rest of the teams. I just format it as a string because it's already in the right visual format